### PR TITLE
James/features/stocks

### PR DESCRIPTION
**Pull request by villanuevajamesfvillanueva**
1. fixed naming of buyer_stocks and broker_stocks tables
2. changed column name of company names in buyer_stocks and broker_stocks
3. added views for common stock lists: most active, gainers, losers
image
4. changed datatype of stock_id to string (the stock symbol will be used as its id in the database)
5. updated notes

### CODE REVIEW

**Review by dlanoronald123 — APPROVED** [review]
aprroved master

File: app/models/broker_stock.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class BrokerStock < ApplicationRecord
   belongs_to :broker
-  validates :name, presence: true
-  validates :quantity, presence: true
-  validates :price, presence: true
+  #   validates :companyname, presence: true
+  #   validates :quantity, presence: true
+  #   validates :price, presence: true
```
Comment (analyn-cajocson) [inline app/models/broker_stock.rb:5]: Remove unnecessary comments

File: app/models/buyer.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 class Buyer < User
-  has_many :buyers_stocks, dependent: :destroy
+  has_many :buyers_stocks, dependent: :nullify
+  #   has_many :buyers_stocks, dependent: :destroy
```
Comment (analyn-cajocson) [inline app/models/buyer.rb:3]: Remove comment

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/models/broker_stock.rb (hunk 1 above)
Comment (villanuevajamesfvillanueva) [reply app/models/broker_stock.rb:5]: Ma'am can we leave this comment here? I just commented it just to make the current feature I'm working on to work. We're gonna need this eventually so, I just left it there.

File: app/models/buyer.rb (hunk 2 above)
Comment (villanuevajamesfvillanueva) [reply app/models/buyer.rb:3]: Please refer to the comment above

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**